helm/defectdojo/values.yaml

```diff
@@ -250,8 +250,14 @@ django:
     # could be emptyDir (not for production) or pvc
     type: emptyDir
     # in case if pvc specified, should point to the already existing pvc
-    persistentVolumeClaim: media
-
+    persistentVolumeClaim:
+      # set to true to create a new pvc and if django.mediaPersistentVolume.type is set to pvc
+      create: false
+      name:
+      size: 5Gi
+      accessModes:
+      - ReadWriteMany
```

Comment:
This is very tricky, so there in values make a note to check doc first  KUBERNETES.md.

Add there information that storage should support readwrite many.

Reply:
Done